RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducerCUDA.cc

```diff
@@ -117,7 +117,8 @@ void PixelVertexProducerCUDA::produce(edm::StreamID streamID, edm::Event& iEvent
     std::cout << "found " << nt << " tracks in cpu SoA for Vertexing at " << tracks << std::endl;
     */
 
-    iEvent.emplace(tokenCPUVertex_, m_gpuAlgo.make(tracks, m_ptMin));
+    cudaStream_t stream=0;
```

Comment:
clang-tidy will complain and suggest to use `nullptr` instead of `0`; as an alternative, I would suggest `cudaStreamDefault` (which is defined to 0)

Reply:
ok, (also elsewhere). will fix.